### PR TITLE
fix empty QueryClauses in association (#5502)

### PR DESCRIPTION
**Pull request by Goxiaoy**
- [x] Do only one thing
- [x] Non breaking API changes
- [x] Tested

### What did this pull request do?

See #5502

### CODE REVIEW

**Comment by a631807682** [comment]
@Goxiaoy Can you add some tests? We need to make sure it won't be broken by other changes.

**Comment by Goxiaoy** [comment]
@a631807682 Added

**Comment by Goxiaoy** [comment]
@a631807682 formated

**Review by a631807682 — APPROVED** [review]

File: tests/associations_test.go [hunk 1]
```diff
@@ -3,6 +3,9 @@ package tests_test
 import (
 	"testing"
 
+	"gorm.io/gorm/clause"
+	"gorm.io/gorm/schema"
+
```
Comment (jinzhu) [inline tests/associations_test.go:8]: Can you delete the newline break and format with gofmt?

Thank you.

**Pull request merged**